apps/example-app/README.md

```diff
@@ -0,0 +1,164 @@
+# Welcome to your new ignited app!
```

Comment:
I recommend customizing this README so people understand it's an Ignited app but exclusively for use to help develop Reactotron.

Reply:
Updated.